jsf/parser.py

```diff
@@ -32,13 +33,16 @@ class JSF:
     def __init__(
         self,
         schema: Dict[str, Any],
-        context: Dict[str, Any] = {
-            "faker": faker,
-            "random": random,
-            "datetime": datetime,
-            "__internal__": {"List": List, "Union": Union, "Tuple": Tuple},
-        },
-        initial_state: Dict[str, Any] = {},
+        context: Dict[str, Any] = MappingProxyType(
```

Comment:
See: [How to Make an Immutable Dict in Python](https://adamj.eu/tech/2022/01/05/how-to-make-immutable-dict-in-python/)